Cargo.toml

```diff
@@ -21,7 +21,12 @@ hf-hub = { version = "0.3.1", features = ["tokio"] }
 [profile.release]
 debug = 1
 incremental = true
+panic = "abort"
```

Comment:
abort is needed to make sure that we crash if a tokio task panics which in our case is not recoverable since have a lot of background tasks for batching validation etc

Reply:
https://github.com/tokio-rs/tokio/issues/2002 Wow. even threads catch panics... Interesting..